aspnetcore/signalr/version-differences.md

```diff
@@ -8,9 +8,19 @@ ms.date: 06/30/2018
 uid: signalr/version-differences
 ---
 
-# Differences between SignalR and ASP.NET Core SignalR
+# Differences between ASP.NET SignalR and ASP.NET Core SignalR
 
-ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET 4.x SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+
+## How to identify which version of SignalR you are using:
+
+|                      | ASP.NET SignalR | ASP.NET Core SignalR |
+| -------------------- | --------------- | -------------------- |
+| Server NuGet Package | [Microsoft.AspNet.SignalR](https://www.nuget.org/packages/Microsoft.AspNet.SignalR/) | [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App/) |
+| Client NuGet Packages | [Microsoft.AspNet.SignalR.Client](https://www.nuget.org/packages/Microsoft.AspNet.SignalR.Client/) [Microsoft.AspNet.SignalR.JS](https://www.nuget.org/packages/Microsoft.AspNet.SignalR.JS/) | [Microsoft.AspNetCore.SignalR.Client](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR.Client/) |
+| Client NPM Package | [signalr](https://www.npmjs.com/package/signalr) | [@aspnet/signalr](https://www.npmjs.com/package/@aspnet/signalr) |
+| Server App Type | ASP.NET (System.Web) or OWIN Self-Host | ASP.NET Core |
+| Supported Server Platforms | .NET Framework 4.5 | .NET Framework 4.6.1<br/>.NET Core 2.1 |
```

Comment:
The `<br/>` here seems a bit sketchy but it works. I'd like to manually break the line here instead of risking weird flow issues.

Reply:
That's fine ... we do it fairly often to control how cells are rendered. However, I recommend `<br>` as we aren't XHTML.

Reply:
`<you><got><it><buddy>`